src/components/CallView/shared/LocalAudioControlButton.vue

```diff
@@ -4,43 +4,85 @@
 -->
 
 <template>
-	<NcPopover ref="popover"
-		:boundary="boundaryElement"
-		:show-triggers="[]"
-		:hide-triggers="['click']"
-		:auto-hide="false"
-		no-focus-trap
-		:shown="popupShown">
-		<template #trigger>
-			<NcButton :title="audioButtonTitle"
-				:variant="variant"
-				:aria-label="audioButtonAriaLabel"
-				:class="{ 'no-audio-available': !model.attributes.audioAvailable }"
-				:disabled="!isAudioAllowed"
-				@click.stop="toggleAudio">
-				<template #icon>
-					<VolumeIndicator :audio-preview-available="model.attributes.audioAvailable"
-						:audio-enabled="showMicrophoneOn"
-						:current-volume="model.attributes.currentVolume"
-						:volume-threshold="model.attributes.volumeThreshold"
-						overlay-muted-color="#888888" />
-				</template>
-			</NcButton>
-		</template>
-		<div class="popover-hint">
-			<span>{{ speakingWhileMutedWarner?.message }}</span>
-		</div>
-	</NcPopover>
+	<div class="local-audio-control-wrapper">
+		<NcPopover ref="popover"
+			:boundary="boundaryElement"
+			:show-triggers="[]"
+			:hide-triggers="['click']"
+			:auto-hide="false"
+			no-focus-trap
+			:shown="popupShown">
+			<template #trigger>
+				<NcButton :title="audioButtonTitle"
+					:variant="variant"
+					:aria-label="audioButtonAriaLabel"
+					:class="{
+						'no-audio-available': !isAudioAvailable,
+						'audio-control-button': showDevices,
+					}"
+					:disabled="!isAudioAllowed"
+					@click.stop="toggleAudio">
+					<template #icon>
+						<VolumeIndicator
+							:audio-preview-available="isAudioAvailable"
+							:audio-enabled="showMicrophoneOn"
+							:current-volume="model.attributes.currentVolume"
+							:volume-threshold="model.attributes.volumeThreshold"
+							overlay-muted-color="#888888" />
+					</template>
+				</NcButton>
+			</template>
+			<div class="popover-hint">
+				<span>{{ speakingWhileMutedWarner?.message }}</span>
+			</div>
+		</NcPopover>
+
+		<NcActions v-if="showDevices"
+			:disabled="!isAudioAvailable || !isAudioAllowed"
+			class="audio-selector-button"
+			@open="updateDevices">
+			<template #icon>
+				<IconChevronUp :size="16" />
+			</template>
+			<NcActionCaption :name="t('spreed', 'Select a microphone')" />
+			<NcActionButton v-for="device in audioInputDevices"
+				:key="device.deviceId ?? 'none'"
+				type="radio"
+				:model-value="audioInputId"
+				:value="device.deviceId"
+				:title="device.label"
+				@click="handleAudioInputIdChange(device.deviceId)">
```

Comment:
This click mutes the microphone, probably the same happening for video. Most likely comes from assumption, that when we're in the call opening device checker, microphone and camera should be disabled on change, as you might want to try different devices before applying settings